specification/hanaonazure/resource-manager/Microsoft.HanaOnAzure/preview/2017-11-03-preview/hanaonazure.json

```diff
@@ -284,6 +284,52 @@
                     }
                 }
             }
+        },
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.HanaOnAzure/hanaInstances/{hanaInstanceName}/monitoring": {
+            "post": {
+                "tags": [
+                    "HanaOnAzure"
+                ],
+                "operationId": "HanaInstances_Monitoring",
```

Comment:
Please use the `Verb_Noun` format for operation id. A better id here would be `Monitor_HanaInstances`

Reply:
That causes a warning with autorest, should I do it anyway?

Reply:
Could you share what the error is

Reply:
```
WARNING (PostOperationIdContainsUrlVerb/R2066/SDKViolation): OperationId should contain the verb: 'monitoring' in:'Monitor_HanaInstances'. Consider updating the operationId
    - file:///Users/pagebowers/repos/azure-rest-api-specs/specification/hanaonazure/resource-manager/Microsoft.HanaOnAzure/preview/2017-11-03-preview/hanaonazure.json:293:16 ($.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.HanaOnAzure/hanaInstances/{hanaInstanceName}/monitoring"].post.operationId)
```

Reply:
I see, you can name it as `Monitoring_HanaInstances` then
On a related note, in general ARM end points should actually look like `get`/`put`, etc